docs/design_docs/index.md

```diff
@@ -0,0 +1,5 @@
+# Design Docs Index
+
+| Name                                        | Link                                                                                                    |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
```

Comment:
would it be worth adding a status label?

That's kind of a cool way to mark it as approved, since it requires a PR to update the status

Reply:
I think the idea I was going for was that inclusion in this list via a merged PR indicates acceptance